Workload-Security/Support/ssm-iam-example-policy.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "ssm:SendCommand",
+                "ssm:ListCommands",
+                "ec2:DescribeInstances",
+                "ssm:DescribeInstanceInformation",
+                "ssm:ListCommandInvocations"
+            ],
+            "Resource": "*"
```

Comment:
Do we really need like this? no other way to filter?

Reply:
I changed it to be more strict. This is just an example policy to reference.